proc_macros/src/grouped_ordering.rs

```diff
@@ -0,0 +1,186 @@
+use inflector::Inflector;
```

Comment:
Extracted the existing macro into a separate file, just the instantiate-macro stuff is new